app/models/item.rb

```diff
@@ -66,4 +66,13 @@ def discount_apply?(quantity)
     end
   end
 
+  def discount_price
+    percent_off = discount.percentage / 100.to_f
+    price * (1 - percent_off).round(2)
+  end
```

Comment:
Can this calculation live the model? Are there ActiveRecord methods for calculations?